Doc: Making a first refactoring PR to rename the schema reference (#6536)
aristanetworks/ansible-avd: .github/ignore-codespell-words.md

- iif
    Used as test data
- inout

>     Key name `route_map_inout` iEOS Config schema

Author: ```suggestion
    Key name `route_map_inout` the EOS Config schema
```

>     Key name `route_map_inout` iEOS Config schema

Reviewer: ```suggestion
    Key name `route_map_inout` in the EOS Config schema
```

- pathc
    Used as test data
- ect

>     Valid value for class_maps.qos.[].ecn in EOS Config schema

Reviewer: ```suggestion
    Valid value for class_maps.qos.[].ecn in the EOS Config schema
```
